### ISSUE TITLE
Plugin hangs and restarts

### ISSUE THREAD

**Issue by Mateysz** [fjhorrillo/homebridge-airzone-cloud#39]
**Describe The Bug:**
I installed the plugin to be able to control my Airzone system (3 zones) via Homebridge and Apple. Once activated the plugin keeps restarting after this step/error:

[AirzoneCloud] Adding new accessory Bedroom (UUID: debd617b-11d2-4972-8cff-69c98af28df8)
TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

"Bedroom" is my 2nd out of 3 zones.
**To Reproduce:**

**Expected behavior:**

**Logs:**

[22.10.2023, 12:02:09] [AirzoneCloud] The installation status was fully received
[22.10.2023, 12:02:09] [AirzoneCloud] Restoring existing accessory from cache: Living room (UUID: 05740c46-3971-4754-9fe0-c05723c56194)
[22.10.2023, 12:02:09] [AirzoneCloud] Adding new accessory Bedroom (UUID: debd617b-11d2-4972-8cff-69c98af28df8)
TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[22.10.2023, 12:02:09] [AirzoneCloud] Child bridge process ended

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by thezepter** [comment]
Same Problem here. Child Bridge restart every 10-20 seconds.

Logs:

`[10/26/2023, 10:10:35 PM] [AirzoneCloud] Child bridge process ended
[10/26/2023, 10:10:35 PM] [AirzoneCloud] Process Ended. Code: 1, Signal: null
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Restarting Process...
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Launched child bridge with PID 29911
[10/26/2023, 10:10:42 PM] Registering platform 'homebridge-airzone-cloud.AirzoneCloud'
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Loaded homebridge-airzone-cloud v0.3.6 child bridge successfully
[10/26/2023, 10:10:42 PM] Loaded 4 cached accessories from cachedAccessories.0E67A12C9136.
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Loading accessory from cache: Badezimmer
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Loading accessory from cache: Schlafen
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Loading accessory from cache: Liam
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Loading accessory from cache: Clara
[10/26/2023, 10:10:42 PM] [AirzoneCloud] Initialice conection to Airzone Cloud
[10/26/2023, 10:10:42 PM] Homebridge v1.6.1 (HAP v0.11.1) (AirzoneCloud) is running on port 33506.
[10/26/2023, 10:10:44 PM] [AirzoneCloud] Logged in successfully as emailadress@mail.com
[10/26/2023, 10:10:44 PM] [AirzoneCloud] Websocket connected
[10/26/2023, 10:10:44 PM] [AirzoneCloud] The installation status was fully received
[10/26/2023, 10:10:44 PM] [AirzoneCloud] Restoring existing accessory from cache: Badezimmer (UUID: 044204b8-0b15-4429-84eb-c030841b6e38)
[10/26/2023, 10:10:44 PM] [AirzoneCloud] Restoring existing accessory from cache: Schlafen (UUID: 4acadfd1-c43f-42c2-ab83-be3ca839d67d)
TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:277:11)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[10/26/2023, 10:10:44 PM] [AirzoneCloud] Child bridge process ended`


- Plugin Version: v0.3.6
- Homebridge Version: [v1.6.1]
- Node.js Version: [v18.13.0]
- NPM Version: ?
- Operating System: Docker / Ubuntu Focal Fossa (20.04.5 LTS)

**Comment by simonetolomelli** [comment]
Very same here.

- Plugin Version: v0.3.6 (but I've tried to downgrade it to version 2.0 and the cycle of death keeps running)
- Homebridge Version: [v1.7.0]
- Operating System: Raspbian GNU/Linux Bullseye (11)
- Node.js Version: 18.16 

```
[12/11/2023, 01:39:06] [AirzoneCloud] Group: Casa<6500b408949380c2dc60db72>
[12/11/2023, 01:39:06] [AirzoneCloud] AirZone Device: Camera<64afe37c76bdfe666f51313e>
[12/11/2023, 01:39:06] [AirzoneCloud] Adding new accessory Camera (UUID: 98ac2085-aa66-4184-a732-940b3e232d05)
[12/11/2023, 01:39:06] [AirzoneCloud] Accesory context: {"device":{"id":"64afe37c76bdfe666f51313e","groupId":"6500b408949380c2dc60db72","installationId":"6500b408949380c2dc60db73","name":"Camera","serialNumber":"28:CC:FF:00:9F:3A","model":"az_zone","firmwareRevision":"4.01","status":{"units":0,"mode":3,"step":{"fah":1,"celsius":0.5},"setpoint_air_stop":{"celsius":21,"fah":70},"setpoint_air_heat":{"celsius":21,"fah":70},"power":true,"local_temp":{"celsius":21.9,"fah":71},"humidity":46}}}
[12/11/2023, 01:39:06] TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[12/11/2023, 01:39:06] Got SIGTERM, shutting down Homebridge...
[12/11/2023, 01:39:11] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[12/11/2023, 01:39:16] [HB Supervisor] Restarting Homebridge...
```

**Comment by fjhorrillo** [comment]
Hi all, I'm working on it but is due to the last version of hombridge over node >18 and I need to upgrade all my systems including the OS, give me time :)

**Comment by fjhorrillo** [comment]
Solved in v0.3.7, tested with node v20.9.0 and Homebridge 1.7.0 which are the last version
